tools/count_gff_features/count_gff_features.xml

```diff
@@ -1,10 +1,11 @@
-<tool id="count_gff_features" name="Count GFF Features" version="0.1">
+<tool id="count_gff_features" name="Count GFF Features" version="0.2">
     <description></description>
     <requirements>
-        <requirement type="package" version="0.7.1">bx-python</requirement>
+        <requirement type="package" version="1.0.0">galaxy-ops</requirement>
```

Comment:
`galaxy-ops` does not contain `galaxy.datatypes.util.gff_util` (yet). Tools in `tool_collections/gops/` contain a copy in `tool_collections/gops/utils/gff_util.py`, but it may make sense to move it to https://github.com/galaxyproject/gops and update all these tools.

Otherwise this can be reverted to `bx-python`, I think.